source/DomainTheory/Basics/Curry.lagda

```diff
@@ -211,6 +213,46 @@ module _ (𝓓 : DCPO {𝓤} {𝓤'})
                y
                p
 
+\end{code}
+
+Added 3 July 2024 by Tom de Jong.
+
+\begin{code}
+
+ -- We introduce abbreviations for readability
```

Comment:
Can this comment be outside the code environment?

Reply:
Sure. I'll make it part of the new PR I'm about to merge.